contracts/CollectionFoundation.sol

```diff
@@ -0,0 +1,47 @@
+pragma solidity ^0.8.0;
+
+import "./freeportParts/MetaTxContext.sol";
+import "./Collection.sol";
+
+/** This is a contract for creating standalone contracts (collections) for users.
+ *
+ */
+contract CollectionFoundation is MetaTxContext  {
```

Comment:
What does Foundation mean? The terms I know for this idea would be "Factory" or "Registry".